episodes/lesson-infrastructure.md

```diff
@@ -24,10 +24,10 @@ By the end of this session, participants should be able to...
 
 :::::::::::::::::::::::::::::::::::::::::::::::::: checklist
 
-## Reading List
+## Reading List & Homework
 
 - [CLDT episode: The Carpentries Workbench](https://carpentries.github.io/lesson-development-training/07-infrastructure.html)
-- FIXME
+  - Please follow the steps in the episode, creating and configuring a new lesson repository, while you read through the content.
```

Comment:
Maybe we should prompt them to note down any issues they had?  They hopefully are already doing that for lesson content but even an issue that doesn't change the curriculum might be good to discuss this week.

Reply:
It is already in the discussion below but just in case they don't read ahead

Reply:
```suggestion
  - Please follow the steps in the episode, creating and configuring a new lesson repository, while you read through the content. 
  - Note down any problems you encounter along the way, no matter whether they were minor (i.e. slowed you down or confused you temporarily, but did not stop you from progressing) or major (i.e. you had to give up or ask for help).
```

Reply:
That change looks great!